easy_my_coop_es/readme/DESCRIPTION.rst

```diff
@@ -1 +1,6 @@
 This addon implements Spanish localization for Easy My Coop
+
+Features:
+
+- Change property_cooperator_account to not accept deprecated account
```

Comment:
This should probably not be done in a localization module (maybe add a comment in ROADMAP.rst ?)